ManatiCR/vitaminau
# VIT-29: Sales of products form

# Task [VIT-29](https://manati.atlassian.net/browse/VIT-29?atlOrigin=eyJpIjoiMDgwNDFkZmVlY2ZhNDUwZTk5NWM2ODI5NTAzMWJmYTIiLCJwIjoiaiJ9)
Creation of the "Venta de productos" form.

#Installation
- Run the following command
`ahoy drush cr && ahoy drush cim -y`

# Test
- Go to the following url: 
  - http://vitaminau.docker/user
- Login as admin user:
  - User: admin
  - Pass: admin
- Go to the following url: 
  - http://vitaminau.docker/form/sale-of-products
- Once on that page, you should be able to see a form that contains all the fields specified in the Jira:
  - [VIT-28](https://manati.atlassian.net/browse/VIT-29?atlOrigin=eyJpIjoiMDgwNDFkZmVlY2ZhNDUwZTk5NWM2ODI5NTAzMWJmYTIiLCJwIjoiaiJ9)
- Fill all the fields of the form and press the save buttom
- Once you press it your information should be saved and the form should redirect you to a thank you page
- Save your draft with the button "Save Draft" and log in again in another browser or incognito, a "Se ha cargado el borrador." message should appear.
- To test the required fields, leave a blank, which has a red "*" and the same form will tell you that it has not been completed yet.

# Acceptance criteria
- Only authenticated users can access the form.
- Form can be saved as draft.
- When the required fields are empty, you should see a message, indicating that it must be filled
- I can complete the form and send it if I complete it correctly.
- The following fields are not required
  - Comentario sobre código CIIU
  - Directa -> Detalle
  - Web -> Detalle
  - Contrato -> Detalle
  - Distribuidores -> Detalle
  - Restricciones
  - ¿Algo que agregar?
  - PDF relevante
  - ¿Cómo calificaría este formulario?
  - ¿Qué mejoraría?
